src/sql/parts/dashboard/common/dashboardWidget.ts

```diff
@@ -41,6 +41,15 @@ export interface TabConfig extends IDashboardTab {
 	actions?: Array<Action>;
 }
 
+export type IUserFriendlyIcon = string | { light: string; dark: string; };
```

Comment:
Nice, glad we have both. Do we fallback correctly if someone provides just light or dark?

Reply:
We is no support on icon just yet

---
In reply to: [170362743](https://github.com/Microsoft/sqlopsstudio/pull/766#discussion_r170362743) [](ancestors = 170362743)